mtp_noms_ops/apps/security/models.py

```diff
@@ -48,7 +48,7 @@ class PaymentMethod(enum.Enum):
 security_check_statuses = {
     'pending': _('To action'),
     'accepted': _('Accepted'),
-    'auto-accepted': _('Auto Accepted'),
+    'auto-accepted': _('Auto accepted'),
```

Comment:
This doesn't actually have an functional effect, as it's only used to display the status of a past check, and it's always coerced into all-caps in a similar way to the other statuses